application/play/java/example/.g8/form/app/controllers/$model__Camel$Controller.java

```diff
@@ -0,0 +1,43 @@
+package controllers;
+
+import play.data.Form;
+import play.data.FormFactory;
+import play.mvc.Controller;
+import play.mvc.Result;
+
+import javax.inject.Inject;
+
+// Add the following to conf/routes 
+/*
+GET     /$model;format="camel"$        controllers.$model;format="Camel"$Controller.$model;format="camel"$Get
+POST    /$model;format="camel"$        controllers.$model;format="Camel"$Controller.$model;format="camel"$Post
+*/
```

Comment:
Is this comment auto generated or is it an action the user has to take?

If it's the latter we should mention it in the README.

If it's not important I'd say we remove it, because it can be misunderstood as a required action due to the phrasing.

Reply:
These files are generated by the template, but they are not used for our case.
Therefore, I removed them and all the files from `.g8` directory